CMakeLists.txt

```diff
@@ -196,4 +184,6 @@ SET_PROPERTY(TARGET ${PROJECT_NAME} PROPERTY FOLDER "plugins")
 # Config files and install rules for pythons scripts
 sofa_install_pythonscripts(PLUGIN_NAME ${PROJECT_NAME} PYTHONSCRIPTS_SOURCE_DIR "python")
 
+sofa_add_subdirectory(plugin extensions/CUDA BeamAdapter.CUDA)
```

Comment:
There should no be a if(SofaCUDA_FOUND) here?
What is the behavior if SofaCUDA is not compiled?

Reply:
This line does not activate the compilation of the plugin. As any plugin, the developer must enable it in cmake.

Reply:
ok so default value is OFF?

Reply:
and if it is set to ON and SofaCUDA is not present, we will have a cmake error, right?

Reply:
It is OFF by default. And you are right about the cmake error.